src/pivot-table/components/icons/Plus.tsx

```diff
@@ -3,20 +3,21 @@ import React from "react";
 import { PLUS_MINUS_ICON_SIZE } from "../../constants";
 
 interface IconProps {
-  onClick: ((e: React.SyntheticEvent<Element, Event>) => void) | undefined;
+  onClick?: ((e: React.SyntheticEvent<Element, Event>) => void) | undefined;
   color: string;
-  testid: string;
   opacity: number;
 }
 
-const PlusIcon = ({ color, opacity, testid, onClick }: IconProps): JSX.Element => (
+export const testIdExpandIcon = "expand-icon";
```

Comment:
has these testids used in test cases? if not then you probably dont need this variable in both files (Minus and Plus component)

Reply:
Yes, the tests are using them to check if the icon exists or not